xunit.runner.visualstudio.desktop/VsTestRunner.cs

```diff
@@ -111,6 +111,9 @@ void ITestExecutor.RunTests(IEnumerable<string> sources, IRunContext runContext,
                 sources = Directory.GetFiles(sourcePath, "*.dll")
                                    .Where(file => !platformAssemblies.Contains(Path.GetFileName(file)))
                                    .ToList();
+
+                ((List<string>)sources).AddRange(Directory.GetFiles(sourcePath, "*.exe")
+                                   .Where(file => !platformAssemblies.Contains(Path.GetFileName(file))));
```

Comment:
that was a weird merge issue...@danstiner rebased and then I checked out...that commit showed up in this branch for some reason and I'm not sure how to make it go away. I should be rebased out.